torch/onnx/symbolic_opset12.py

```diff
@@ -47,16 +48,29 @@ def outer(g, input, other):
     return einsum_helper(g, "i,j->ij", [input, other])
 
 
-@symbolic_helper.parse_args("v", "f", "i")
-def dropout(g, input, p, train):
+def _dropout_returns_masked_input_and_mask(
+    g, input: torch._C.Value, p: float, train: bool
+) -> Tuple[torch._C.Value, Optional[torch._C.Value]]:
     symbolic_helper.check_training_mode(train, "dropout")
-    # if train is False, dropout is no-op
+    # In eval mode, dropout is non-op. That is, if the node's
+    # train param is set to False, dropout just returns its inputs.
     if not train:
-        return input
+        return input, None
     p = g.op("Constant", value_t=torch.tensor(p))
     t = g.op("Constant", value_t=torch.tensor(train, dtype=torch.bool))
-    r, _ = g.op("Dropout", input, p, t, outputs=2)
-    return r
+    r, mask = g.op("Dropout", input, p, t, outputs=2)
+    return r, mask
+
+
+@symbolic_helper.parse_args("v", "f", "i")
+def dropout(g, input, p, train):
+    masked, _ = _dropout_returns_masked_input_and_mask(g, input, p, train)
+    return masked
+
+
+@symbolic_helper.parse_args("v", "f", "i")
+def native_dropout(g, input, p, train):
```

Comment:
While you are at it, please help us type annotate these two symbolic functions as well. thanks!